pubsub/gossipsub/gossipsub-v1.1.md

```diff
@@ -470,6 +471,20 @@ if invalidMessageDeliveries < DecayToZero {
 
 `TBD`: We are currently developing multiple types of simulations that will inform us on how to best recommend tunning the Scoring function. We will update this section once that work is complete
 
+#### Extended Validators
+
+The pubsub subsystem incorporates application-specific message validators so that the application can
+signal invalid message delivery, and trigger the P₄ penalty.
+However, It is possible to have circumstances where a message should not be delivered to the application
+or forwarded to the network, but without triggering the P₄ penalty.
+A known use-case where this need exists is in the case of duplicate beacon messages or while an application is syncing its blockchain, in which case it would be unable to ascertain the validity of new messages.
+
+In order to address this situation, all gossipsub v1.1 implementations _must_ support extended validators with a ternary decision interface.
```

Comment:
I'd prefer to keep the current cardinality out of the spec -- because it makes this a bit bespoke. I'd support making this an enum, out of which three values are supported in this version of the spec.

Reply:
Yes, indeed this is the intention as well, will update.

Reply:
done, I called it "enumerated interface".